internal/server/evaluation/evaluation_server.go

```diff
@@ -0,0 +1,30 @@
+package evaluation
+
+import (
+	"go.flipt.io/flipt/internal/storage"
+	rpcEvalution "go.flipt.io/flipt/rpc/flipt/evaluation"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+// EvaluateServer serves the Flipt evaluate v2 gRPC Server.
+type EvaluateServer struct {
+	logger    *zap.Logger
+	store     storage.Store
+	evaluator MultiVariateEvaluator
+	rpcEvalution.UnimplementedEvaluationServiceServer
+}
+
+// NewEvaluateServer is constructs a new EvaluateServer.
+func NewEvaluateServer(logger *zap.Logger, store storage.Store) *EvaluateServer {
+	return &EvaluateServer{
+		logger:    logger,
+		store:     store,
+		evaluator: NewEvaluator(logger, store),
+	}
+}
```

Comment:
Any thoughts and feels on just calling this `Server` and `NewServer` (or even just `New` as it is nested in the `server` package?

Just thinking eveluation is in the package name.